stubs/model.py

```diff
@@ -1666,13 +1631,13 @@ def monolithic_solve(self):
                         "attempt_timestep_restart_on_divergence is False. Exiting."
```

Comment:
We should consider adding the "failed to converge" region to this runtime error message, I think. Perhaps ...f"Model {self.name}: SNES diverged (Reason = {self.solver.getConvergedReason()}) and "...

Reply:
Fixed in [f8eaa1a](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/f8eaa1aaec9d83148ef6dac5741a4913832d799b)